services/director-v2/src/simcore_service_director_v2/modules/db/repositories/users.py

```diff
@@ -1,11 +1,18 @@
 from models_library.users import UserID
+from pydantic import EmailStr, parse_obj_as
+from simcore_postgres_database.models.users import UserRole
 from simcore_postgres_database.utils_users import UsersRepo
 
 from ._base import BaseRepository
 
 
 class UsersRepository(BaseRepository):
-    async def get_user_email(self, user_id: UserID) -> str:
+    async def get_user_email(self, user_id: UserID) -> EmailStr:
         async with self.db_engine.acquire() as conn:
-            email: str = await UsersRepo.get_email(conn, user_id)
-            return email
+            email = await UsersRepo.get_email(conn, user_id)
+            return parse_obj_as(EmailStr, email)
+
+    async def get_user_role(self, user_id: UserID) -> UserRole:
+        async with self.db_engine.acquire() as conn:
+            user_role = await UsersRepo().get_role(conn, user_id=user_id)
+            return UserRole(user_role)
```

Comment:
this comes from from simcore_postgres_database.models.users import UserRole
...